ga4gh/backends.py

```diff
@@ -365,18 +365,23 @@ class Backend(object):
     """
     Superclass of GA4GH protocol backends.
     """
-    def __init__(self, dataDir, Dataset):
+
+    def __init__(self, dataDir, VariantSet):
+        # We will use dataDir as the Dataset Id.
         self._dataDir = dataDir
-        self._variantSets = {}
-        # All files in datadir are assumed to correspond to Datasets.
-        for vsid in os.listdir(self._dataDir):
-            f = os.path.join(self._dataDir, vsid)
-            self._variantSets[vsid] = Dataset(vsid, f)
-        self._variantSetIds = sorted(self._variantSets.keys())
+        self._variantSetDict = {}
```

Comment:
``self._variantSetMap`` perhaps? We've used ``_tabixMap`` above, so we should probably settle on one convention if we're going to encode some type hints in the variable name. Should we convey that we're mapping ids to objects in there (i.e. ``self._variantSetIdMap``)? This might be a good convention to adopt to make it clear when we're mapping an id to an object?